index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8f283d01ad.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Daron Wiafe Software Developer Portfolio</title>
+</head>
+    <body>
+        <header id="header" class="intro">
+            <a href="#" class="home_link">Daron Wiafe</a>
+            <nav class="portfolio_nav mobile_nav">
+                <ul>
+                    <li><button id="menuButton" type="button" class="menu_btn"><i class="fas fa-bars fa-2x"></i></button></li>
+                    <li class="links"><a href="#portfolio">Portfolio</a></li>
+                    <li class="links"><a href="#about">About</a></li>
+                    <li class="links"><a href="#contact">Contact</a></li>
+                    <li class="links"><i class="fa-solid fa-envelope"></i></li>
+                </ul>
+            </nav>
+        </header>
+        <div id="menu" class="mobile_menu">
+            <div class="position_btn">
+                <button id="cancelButton" type="button" class="mobile_menu_btn"><img src="images/Icon.png" alt="cancel icon"></button>
+            </div>
+            <ul class="mobile_menu_list">
+                <li class="mobile_links"><strong>Portfolio</strong><img class="chevron" src="images/Union.png" alt="chevron"></li>
+                <li class="mobile_links"><strong>About</strong><img class="chevron" src="images/Union.png" alt="chevron"></li>
+                <li class="mobile_links"><strong>Contact</strong><img class="chevron" src="images/Union.png" alt="chevron"></li>
+            </ul>
```

Comment:
Great job! You updated the mobile menu options click:grinning:. The mobile menu works well by hiding when a menu option is clicked. However, there is an issue where clicking on a menu option doesn't take the user to the correct link.:point_up_2:

For example, when clicking on "Contact Me," it should navigate to the corresponding "Contact Me" section, but it currently doesn't. It's important to fix this issue to provide a smoother and easier browsing experience for mobile users.:+1: